src/components/search-form/components/packages-filter/packages-filter.test.js

```diff
@@ -0,0 +1,120 @@
+import { render } from '@testing-library/react';
+import PackagesFilter from './packages-filter';
+import PackagesFilterAccordion from '../packages-filter-accordion';
+
+jest.mock('../packages-filter-accordion', () => jest.fn(() => <div>PackagesFilterAccordion</div>));
+
+const titlesFacets = {
+  packages: [{
+    id: '4591',
+    name: 'name1',
+    count: 100,
+  }, {
+    id: '5478',
+    name: 'name2',
+    count: 200,
+  }],
+};
+
+const activeFilters = { packageIds: ['5478', '4591'] };
+
+const packagesFilterMap = {
+  4591: { id: '4591', name: 'name1', count: 100 },
+  5478: { id: '5478', name: 'name2', count: 200 },
+};
+
+const getComponent = (props = {}) => (
+  <PackagesFilter
+    activeFilters={{ packageIds: '' }}
+    disabled={false}
+    params={{ q: 'book' }}
+    titlesFacets={{}}
+    packagesFilterMap={{}}
+    results={{}}
+    onUpdate={() => {}}
+    {...props}
+  />
+);
+
+const renderComponent = (props = {}) => render(getComponent(props));
+
+describe('Given PackagesFilter', () => {
```

Comment:
Please add some tests to check that `onUpdate` is called with correct data when selecting options